.github/workflows/ci.yml

```diff
@@ -110,3 +112,38 @@ jobs:
         timeout_minutes: 15
         max_attempts: 2
         command: ./tools/test_monitor/run-tests.sh
+  flow-legacy-client-compatibility-test:
+    name: Client Compatibility Tests With Observer
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version:
+          - 1.16
+          - 1.17
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version}}
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Build relic
+      run: make crypto_setup_gopath
+    - name: Docker build
+      run: make install-tools
+    - name: Install Flow Client
+      run: sh -ci "$(curl -fsSL https://storage.googleapis.com/flow-cli/install.sh)" && mv ~/.local/bin/flow /tmp
```

Comment:
we don't really need the `mv` here, we are running github actions on github VMs

Reply:
Indeed. Thank you for spotting this. It is cleaner to call the extracted local version `~/.local/bin/flow` I updated the PR.

Some context: Normally I like to have a fixed path to pick such items within an isolated VM and docker container. However, we cannot predict future environments, the least amount of changes, the less errors we can expect.

Reply:
The update for reference: https://github.com/GetElastech/flow-go/commit/d1aa6d09f949a1c3d3d59280c617720f4bab3fa5